lib/beacon_api/controllers/v1/beacon_controller.ex

```diff
@@ -3,13 +3,14 @@ defmodule BeaconApi.V1.BeaconController do
   alias BeaconApi.ErrorController
 
   alias LambdaEthereumConsensus.ForkChoice
+  alias LambdaEthereumConsensus.Store.Blocks
   alias LambdaEthereumConsensus.Store.BlockStore
```

Comment:
What is the difference between `LambdaEthereumConsensus.Store.Blocks` and `LambdaEthereumConsensus.Store.BlockStore`?

Reply:
`Blocks` is the cache. We should rename these 😅

Reply:
I opened #686 for that

Reply:
nice!